docs/excel/custom-functions-architecture.md

```diff
@@ -0,0 +1,42 @@
+---
+ms.date: 03/06/2019
+description: Learn about Excel custom functions' runtime. 
+title: Custom functions architecture (preview)
+localization_priority: Priority
+---
+# Custom functions architecture
+
+Custom function use a new JavaScript runtime that differs from the browser-based JavaScript engine which powers most other parts of your add-in.
+
+## Custom functions runtime
+
+The custom functions runtime prioritizes executing calculations, so functions run run smoothly when Excel recalculates. The custom functions runtime allows access to APIs which make requesting external data and exchanging data over a persistent connection over a server possible.
+
+Because functions are limited to actions performed in a workbook, the custom functions runtime doesn't support UI elements nor does it provide access to the Document Object Model (DOM). You should think of custom functions as a separate component of your add-in, separate from your add-in's task pane or other UI elements.
+
+>[!NOTE] 
+> When first installing a custom function, a task pane is shown. This pane is necessary to install the custom functions runtime, but shouldn't serve as your main task pane for your add-in.
+
+## Browser-based engine
+
+You may be used to the common browser-based engine which powers most add-ins if you have developed an add-in before. This engine allows access to the Office.js APIs. Any of the Excel APIs (such as APIs which allow you to manipulate Excel tables, charts, etc) you may have used when developing an Excel add-in run on this browser-based engine, but aren't directly accessible from the custom functions runtime. 
```

Comment:
I would just phrase more directly as "The task pane and other components outside of custom functions run in a browser engine runtime. The browser engine runtime enables access to the Office.js APIs..."